soroban-token-sdk/src/tests/events.rs

```diff
@@ -42,10 +42,12 @@ fn test_approve() {
     let token_events = env.events().all();
     assert_eq!(
         token_events,
-        vec![
+        std::vec![ContractEvent::new(
```

Comment:
If we made it possible to compare a Soroban vec with a listed vec, would the need for this change disappear? I'm thinking about whether that change worked, and if it would soften the breaking nature of this change.